### PR TITLE
539 migrate online page

### PR DESCRIPTION
**Pull request by Midnite1212**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,329 @@
+{
```
Comment (albertparedandan) [inline package-lock.json:1]: why is this changed?

Comment (albertparedandan) [reply package-lock.json:1]: same question for all the package.json and yarn.lock files

File: ui/env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_GOOGLE_CLIENT_ID=694013380015-j2lgv2uucaqs6shl72km6unmq77v74fn.apps.googleusercontent.com
```
Comment (albertparedandan) [inline ui/env:1]: why is this changed?

File: server/config/routes.js [hunk 3]
```diff
@@ -40,6 +40,11 @@ module.exports.routes = {
   // Pages APIs
   'get /api/pages/get-pages': 'pages/get-pages',
 
+  //Live Semron APIs
+  'post /api/live-sermons/create-live-sermon': 'liveSermon/create-live-sermon',
```
Comment (albertparedandan) [inline server/config/routes.js:44]: can we call this live-sermon instead of live-sermons? keeping in consistent

Comment (albertparedandan) [reply server/config/routes.js:44]: same comment for the others

File: server/api/controllers/liveSermon/create-live-sermon.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    sermonPassage: {
+      type: 'string',
+    },
+    sermonDescription: {
+      type: 'string',
+    },
+    isPublished: {
+      type: 'boolean',
+    },
+    lastUpdatedBy: {
```
Comment (albertparedandan) [inline server/api/controllers/liveSermon/create-live-sermon.js:38]: i would remove this. and just always assign lastUpdatedBy in your function implementation

File: server/api/controllers/liveSermon/update-live-sermon.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    sermonPassage: {
+      type: 'string',
+    },
+    sermonDescription: {
+      type: 'string',
+    },
+    isPublished: {
+      type: 'boolean',
+    },
+    lastUpdatedBy: {
```
Comment (albertparedandan) [inline server/api/controllers/liveSermon/update-live-sermon.js:42]: similar comment on create-live-sermon. i would remove this

File: server/api/controllers/liveSermon/update-live-sermon.js [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+        sermonDate,
+        sermonPassage,
+        sermonDescription,
+        isPublished,
+        lastUpdatedBy,
+        isDeleted,
+      });
+
+      if (!existing) {
+        throw `Name: ${title}, live sermon doesn't exist`;
```
Comment (albertparedandan) [inline server/api/controllers/liveSermon/update-live-sermon.js:111]: would show the sermon id too in the logs

**Comment by albertparedandan** [comment]
there's a lot of commits here that do not belong. can you remove them?

**Review by albertparedandan — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (Midnite1212) [reply package-lock.json:1]: i think it got updated to the latest version when I do yanr install. should I revert it?

File: ui/env (hunk 2 above)
Comment (Midnite1212) [reply ui/env:1]: eh.. why is the .env uploaded, it should be in the .gitignore? ?

File: package-lock.json (hunk 1 above)
Comment (albertparedandan) [reply package-lock.json:1]: yes pls

**Comment by albertparedandan** [comment]
What happened to isPublished variable? I think it's useful so we should keep it

**Comment by albertparedandan** [comment]
Also, please resolve comments so I know which ones you addressed

**Review by albertparedandan — CHANGES_REQUESTED** [review]
See comments

**Comment by Midnite1212** [comment]
ohh I thought in the prev comments u meant for last updated and isPublished, my bad will add it back

**Review by albertparedandan — APPROVED** [review]
